src/features/withCustomProvider.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { BrowserRouter } from 'react-router-dom';
+import { BrowserRouter } from 'react-router-dom-v5-compat';
```

Comment:
Moet je hier niet CompatRouter gebruiken? https://github.com/remix-run/react-router/discussions/8753

Reply:
I figured I didn't since we want to use the v6 API throughout this package